src/main/resources/bodyRequest.json

```diff
@@ -17,6 +17,47 @@
   "propertyTypeFarm": false,
   "propertyTypeCondo": false,
   "propertyTypeTownHouse": false,
+  "firePlaceFuelType": "fire",
+  "firePlaceDetails": "details",
+  "floorsHardwood": "wood",
+  "floorsHardwoodIndicator": "NO",
+  "displayFlagListing": true,
+  "displayFlagAddress": true,
+  "lotSizeRange": "size",
+  "lotSizeRangeUnits": "SQ_METERS",
+  "lotSizeWidthUnits": "METERS",
+  "diningRoomWidthUnits": "METERS",
+  "familyRoomWidthUnits": "METERS",
+  "livingRoomWidthUnits": "METERS",
+  "basementWidthUnits": "METERS",
+  "lotSizeLengthUnits": "METERS",
+  "diningRoomLengthUnits": "METERS",
+  "livingRoomLengthUnits": "METERS",
+  "familyRoomLengthUnits": "METERS",
+  "basementLengthUnits": "METERS",
+  "parkingTotal": 0,
+  "parkingTotalOperator": "DEFAULT_ENUM_VALUE",
+  "firePlaceFuelType": "fire",
```

Comment:
I see everything here double, is it necessary?

Reply:
oh no, it's because of the conflicts, I'll fix it, thanks